### PR TITLE
add test folder for tests in jest config

### PR DESCRIPTION
**Pull request by ricardo-devis-agullo**

### CODE REVIEW

File: utils/createJestConfig.js [hunk 1]
```diff
@@ -14,16 +14,18 @@ module.exports = (resolve, rootDir, isEjecting) => {
     : undefined;
 
   const config = {
-    roots: ['<rootDir>/src'],
+    roots: ['<rootDir>/src', '<rootDir>/test', '<rootDir>/tests'],
 
-    collectCoverageFrom: ['src/**/*.{js,jsx,ts,tsx}', '!src/**/*.d.ts'],
+    collectCoverageFrom: ['src/**/*.{js,jsx,ts,tsx}', 'tests?/**/*.{js,jsx,ts,tsx}','!src/**/*.d.ts'],
```
Comment (fbarbare) [inline utils/createJestConfig.js]: Should you have something like "!tests?/**/*.d.ts'"?

File: utils/createJestConfig.js [hunk 2]
```diff
@@ -14,16 +14,18 @@ module.exports = (resolve, rootDir, isEjecting) => {
...
 
     setupFiles: [require.resolve('react-app-polyfill/jsdom')],
 
     setupFilesAfterEnv: setupTestsFile ? [setupTestsFile] : [],
     testMatch: [
       '<rootDir>/src/**/__tests__/**/*.{js,jsx,ts,tsx}',
-      '<rootDir>/src/**/*.{spec,test}.{js,jsx,ts,tsx}'
+      '<rootDir>/src/**/*.{spec,test}.{js,jsx,ts,tsx}',
+      '<rootDir>/tests?/**/__tests__/**/*.{js,jsx,ts,tsx}',
+      '<rootDir>/tests?/**/*.{spec,test}.{js,jsx,ts,tsx}',
```
Comment (fbarbare) [inline utils/createJestConfig.js]: I don't think we need all that here:
```
'<rootDir>/tests?/**/__tests__/**/*.{js,jsx,ts,tsx}',
'<rootDir>/tests?/**/*.{spec,test}.{js,jsx,ts,tsx}'
```

`'<rootDir>/tests?/**/*.{js,jsx,ts,tsx}',` is most probably enough as you're already in a test folder

File: utils/createJestConfig.js [hunk 3]
```diff
@@ -14,16 +14,18 @@ module.exports = (resolve, rootDir, isEjecting) => {
...
+    collectCoverageFrom: ['src/**/*.{js,jsx,ts,tsx}', 'tests?/**/*.{js,jsx,ts,tsx}','!src/**/*.d.ts', '!tests?/**/*.d.ts'],
 
     setupFiles: [require.resolve('react-app-polyfill/jsdom')],
 
     setupFilesAfterEnv: setupTestsFile ? [setupTestsFile] : [],
     testMatch: [
       '<rootDir>/src/**/__tests__/**/*.{js,jsx,ts,tsx}',
-      '<rootDir>/src/**/*.{spec,test}.{js,jsx,ts,tsx}'
+      '<rootDir>/src/**/*.{spec,test}.{js,jsx,ts,tsx}',
+      '<rootDir>/tests?/**/__tests__/**/*.{js,jsx,ts,tsx}',
```
Comment (fbarbare) [inline utils/createJestConfig.js]: I guess you can remove that one then

File: utils/createJestConfig.js (hunk 2 above)
Comment (ricardo-devis-agullo) [reply utils/createJestConfig.js]: done

File: utils/createJestConfig.js (hunk 1 above)
Comment (ricardo-devis-agullo) [reply utils/createJestConfig.js]: done

File: utils/createJestConfig.js (hunk 3 above)
Comment (ricardo-devis-agullo) [reply utils/createJestConfig.js]: done

**Comment by fbarbare** [comment]
Would be good if we would allow the "setupTests" to be in the `tests?` folder too

**Pull request merged**